core/src/components/item/item.scss

```diff
@@ -125,11 +130,11 @@ button, a {
 .item-inner {
   @include margin(0);
   @include padding(
-    var(--inner-padding-top),
-    calc(var(--ion-safe-area-right, 0px) + var(--inner-padding-end)),
-    var(--inner-padding-bottom),
-    calc(var(--ion-safe-area-left, 0px) + var(--inner-padding-start))
-  );
+     var(--inner-padding-top),
+     calc(var(--ion-safe-area-right, 0px) + var(--inner-padding-end)),
+     var(--inner-padding-bottom),
+     calc(var(--ion-safe-area-left, 0px) + var(--inner-padding-start))
+   );
```

Comment:
3 spaces indentation does not seem right. maybe all changes in this 5 lines can be reverted